lib/https.js

```diff
@@ -31,9 +31,15 @@ const inherits = util.inherits;
 const debug = util.debuglog('https');
 const { urlToOptions, searchParamsSymbol } = require('internal/url');
 
-function Server(opts, requestListener) {
+function Server(opts = {}, requestListener) {
```

Comment:
Note that using default values is *potentially* a breaking change given that it changes the value of `Server.length`